continue-reading.html

```diff
@@ -1,13 +1,155 @@
+      <p7>The first was to never eat blue food. There was nothing in nature that was edible that was blue. People often
+        asked about blueberries, but everyone knows those are.</p7>
+      <p8> actually purple. Stranded. Yes, she was now the first person ever to land on Venus, but that was of little
+        consequence. Her name would be read by millions in school as the first to land here, but that celebrity would
+        never actually be seen by her.</p8>
+      <a href="learnmore.html" class="learnsomethings">Learn Here Click</a>
+      <hr class="chicagoline">
+      <h8>You Shouldnt Still Be Reading</h8>
+      <img class="smoll" img
+        src="https://resizing.flixster.com/JGMa1gx3JGu7MVkFGwm4AsTQ16c=/206x305/v1.bTsxMTIwNDIyNjtqOzE4NjI0OzEyMDA7MTIwMDsxNjAw">
+      <p9>If you can imagine a furry humanoid seven feet tall, with the face of an intelligent gorilla and the braincase
+        of a man, you'll have a rough idea of what they looked like -- except for their teeth.The canines would</p9>
+      <p10>have fitted better in the face of a tiger, and showed at the corners of their wide, thin-lipped mouths,
+        giving them an expression of ferocity.</p10>
+      <a href="learnmore.html" class="learnsomethings2">Learn MOAR</a>
+    </div>

```

Comment:
On my screen this section is going through the middle of your page and on top of everything. This could be because of the way some of your previous CSS is forcing your content to move around